src/main/java/com/google/cloud/resourcemanager/OrgPolicyInfo.java

```diff
@@ -0,0 +1,397 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.resourcemanager;
+
+import com.google.api.services.cloudresourcemanager.model.OrgPolicy;
+import com.google.api.services.cloudresourcemanager.model.RestoreDefault;
+import com.google.common.base.Function;
+import com.google.common.base.MoreObjects;
+import java.io.Serializable;
+import java.util.List;
+import java.util.Objects;
+
+/** A Google Cloud Resource Manager organization policy metadata object. */
+public class OrgPolicyInfo implements Serializable {
+  static final Function<OrgPolicy, OrgPolicyInfo> FROM_PB_FUNCTION =
+      new Function<com.google.api.services.cloudresourcemanager.model.OrgPolicy, OrgPolicyInfo>() {
+        @Override
+        public OrgPolicyInfo apply(
+            com.google.api.services.cloudresourcemanager.model.OrgPolicy pb) {
```

Comment:
you've imported this; I don't think you need the long name

Reply:
done